packages/react-hooks/src/useMergedRefs.ts

```diff
@@ -1,22 +1,28 @@
-import { useCallback, Ref, MutableRefObject } from 'react';
+import * as React from 'react';
+
 /**
  * React hook to merge multiple React refs (either MutableRefObjects or ref callbacks) into a single ref callback that
  * updates all provided refs
  * @param refs- Refs to collectively update with one ref value.
  */
-export function useMergedRefs<T>(...refs: (Ref<T> | undefined)[]): (instance: T) => void {
-  return useCallback(
+export function useMergedRefs<T>(...refs: (React.Ref<T> | undefined)[]): ((value: T) => void) & React.RefObject<T> {
```

Comment:
Would be good to add a `@returns` comment explaining the return value

Reply:
done